docs/src/modules/concepts/pages/state-model.adoc

```diff
@@ -62,7 +60,7 @@ Event Sourced entities are replicated between all regions in an Akka project by
 
 In order to have multi-writer (or write anywhere) capabilities you must implement a conflict-free replicated data type (CRDT) for your Event Sourced Entity. This allows data to be shared across multiple instances of an entity and is eventually consistent to provide high availability with low latency. The underlying CRDT semantics allow replicated Event Sourced Entity instances to update their state independently and concurrently and without coordination. The state changes will always converge without conflicts, but note that with the state being eventually consistent, reading the current data may return an out-of-date value.
 
-Key Value Entities do not replicate state between regions at this time, but will soon. When they do they will use a Last Writer Wins (LWW) reconciliation strategy. 
+NOTE: Although Key Value Entities are planned to support a Last Writer Wins (LWW) mechanism, this feature is not yet available.
```

Comment:
Should this caveat be dropped now?

Reply:
That is about multi-writer conflict resolution. That is still not implemented.